tests/safety/test.c

```diff
@@ -36,6 +36,7 @@ TIM_TypeDef *TIM2 = &timer;
 #define HW_TYPE_BLACK_PANDA 3U
 #define HW_TYPE_PEDAL 4U
 #define HW_TYPE_UNO 5U
+#define HW_TYPE_DOS 6U
```

Comment:
good catch!

Reply:
Also need to add to python lib, just a note for myself